### ISSUE TITLE
jsDom peerDependency precludes two major versions

### ISSUE THREAD

**Issue by jjoekoullas** [badeball/karma-jsdom-launcher#31]
Noticed this when jsdom was not detecting canvas was installed alongside it.  Updating to jsdom to 15 fixed it, however this package only lists up to 13 as a valid peer dependency.

I've updated the peer dependencies to 15 for my own use, but due #30 I'm not able to run tests.

**Comment by badeball** [comment]
Support for v14 and v15 has been added with v7.1.1. Thanks for letting me know.

**Comment by badeball** [comment]
I've "watched" the JSDOM repository for new releases for some time now, but never actually received any notifications. I've registered that [1-2] even though publishing tags is listed as a "release" in the interface, it's not considered notification-worthy unless it is accompanied by a description (lol, what kind of a joke is this). I've subscribed to newreleases.io, which hopefully will let me add new support in a more timely manner. 

[1] https://github.com/isaacs/github/issues/410
[2] https://github.community/t5/How-to-use-Git-and-GitHub/Not-getting-release-notification-despite-quot-Watching-releases/td-p/19665
